enhancements/api-review/azure_user_defined_tags.md

```diff
@@ -0,0 +1,322 @@
+---
+title: azure_user_defined_tags
+authors:
+  - "@bhb"
+reviewers:
+  - "@patrickdillon"
+  - "@sdodson"
+  - "@jhixson74"
+  - "@JoelSpeed"
+  - "@dhellmann"
+  - "@jewzaam"
+  - "@adambkaplan"
+  - "@abhat"
+  - "@dmage"
+  - "@Miciah"
+  - "@abutcher"
+  - "@staebler"
+  - "@jerpeter1"
```

Comment:
this is a lot of people you've asked to review this which:

1) will consume a lot of people's time
2) will make it difficult to reach consensus and move forward

Can you please provide a sentence for each person indicating why you are requesting their review/what aspects of this EP they should be focusing on?

(if someone of these people are only listed for their awareness, i'd just tag/point them to the EP, not list them as reviewers)

Reply:
Yeah I agree, will comment with a reason for adding as reviewers and will CC others.